README.md

```diff
@@ -50,13 +53,16 @@ _Note that we are making this request directly to Elasticsearch (port 9200) so w
 Let's get all the documents belonging to `tenant-1` and see what's in the index!
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\"" localhost:9200/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1" localhost:9200/try_cloaked_search/_search | jq
```

Comment:
This change won't work once the improved tenant detection PR lands. Any tenant id which isn't entirely alphanumeric must be quoted.